src/user-statuses/__tests__/userStatusReducer-test.js

```diff
@@ -0,0 +1,117 @@
+/* @flow strict-local */
```

Comment:
This file's basename (as well as the directory above it) should probably change as part of the rename:
`src/user-statuses/__tests__/userStatusReducer-test.js`